elasticsearch/kibana/init.sls

```diff
@@ -79,6 +79,34 @@ remove_default_nginx_config:
   file.absent:
     - name: {{ kibana.nginx_site_path }}/default
 
+{% if salt.grains.get('init') == 'systemd' %}
+add_node_environment_variables:
+  file.managed:
+    - name: /etc/systemd/system/kibana.service.d/kibana_env.conf
+    - makedirs: True
+    - contents: |
+        [Service]
+        {% for env in kibana.kibana_env %}
+        Environment='{{ env }}'
+        {% endfor %}
```

Comment:
It seems like these declarations would also be appropriate here:

```
[Unit]
Description=Kibana Analytics Dashboard
After=network.target

[Install]
WantedBy=multi-user.target
```

Description is just a nit for viewing with `systemctl`.